docs/source/building.rst

```diff
@@ -333,3 +333,46 @@ your fellow users you can install the package in a local directory:
 
  cd SimpleITK-build/Wrapping/R/Packaging
  R CMD INSTALL -l /path_to/my_R_libs SimpleITK
+
+
+Building SimpleITK documentation
+================================
+
+SimpleITK uses `Sphinx <https://www.sphinx-doc.org>`__ to build documentation. The Sphinx build system is not integrated with the CMake configuration and build infrastructure, so user need to set it up separately, and then use it to build SimpleITK documentation.
```

Comment:
Replace above text with the more specific information:
SimpleITK uses `Sphinx <https://www.sphinx-doc.org>`__ to create the documentation hosted on read-the-docs. To build this documentation we set up a Python environment with the appropriate packages.